### ISSUE TITLE
Double slash

### ISSUE THREAD

**Issue by beckindesigns** [magento/magento2#2149]
During the install there is a double slash for the store url. Example: http://yoursite.com//magento2/

**Comment by mazhalai** [comment]
@beckindesigns was this using Web Setup Wizard or CLI?

**Comment by beckindesigns** [comment]
@mazhalai sorry, that occurred during the Web Setup Wizard. Thanks!

**Comment by mazhalai** [comment]
@beckindesigns we are unable to reproduce this. What is your environment and how did you deploy: git clone or composer create-project?

**Comment by beckindesigns** [comment]
I used git clone to pull in the files. It's definitely there. Would you like me to do another installation and record my process?

**Comment by mazhalai** [comment]
Yes, please mention what changes were made. For example, if default store address was changed etc.

**Comment by beckindesigns** [comment]
No problem at all, I'll do that. I'll take a screen recording of the process for you. Thanks!

**Comment by beckindesigns** [comment]
Here you go. https://vimeo.com/143163215 Excuse my southern Bama voice lol

**Comment by mazhalai** [comment]
@beckindesigns thank you for the video! Can you please give more details about your environment? Also, if you are on `master` or `develop` branch

**Comment by beckindesigns** [comment]
Hi @mazhalai I am running Apache, PHP-FPM, PHP Cache - Zend Cache, Percona Database Server & Memcache on a dedicated server. I am also pulling directly from Master. Let me know if that helps.

Thanks!

**Comment by daim2k5** [comment]
Hi @beckindesigns thanks again for the great video. Can you try the `develop` branch?

**Comment by beckindesigns** [comment]
Sure @daim2k5 I'll give it a shot and let you know how it goes.

**Comment by daim2k5** [comment]
@beckindesigns any update on this? Or can we close this issue?

**Comment by jkristoffer** [comment]
I'm facing the same issue as described here.

Im using develop branch `commit 3991d65ea5b4c91b39f130ceadac7e413233ada6`. I installed using the CLI with `--base-url={{base_url}}` and all the links have a double slash at the end of the base url.

I tried omitting `--base-url` completely and the double slashes disappeared. However i'm running magento2 in a docker container and the server rewrites the base url to `0.0.0.0`.
